index.js

```diff
@@ -0,0 +1,132 @@
+const inquirer = require("inquirer");
+const Manager = require("./src/Manager");
```

Comment:
nice job importing all external modules and classes at the top of your file, great for readability! :-)